projects/components/src/filtering/filter/parser/parsed-filter.ts

```diff
@@ -0,0 +1,13 @@
+import { FilterOperator } from '../filter-operators';
+
+export interface ParsedFilter<TValue> {
+  field: string;
+  operator: FilterOperator;
+  value: TValue;
+}
+
+export interface SplitFilter<TOperator extends string> {
+  lhs: string;
```

Comment:
What's the thinking on how this will support a lhs with a dotted path for CONTAINS_KEY_VALUE?

Reply:
I have the following PR ready. Its not much beyond what is here, but since its done its probably easier to just show you the code.